src/components/ButtonPanel.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Button from './Button';
+
+
+class ButtonPanel extends React.Component {
+constructor () {
+    super();
+    this.state = {
+      panelElm : ['AC', '+/-', '%', '+',
+                  '7', '8', '9', 'X',
+                  '4', '5', '6', '-',
+                  '1', '2', '3', '+',
+                  '0', '.', '','=']
+    }
+  }	
+
+  render() {
+    return (
+      <div className='buttonPanel'>
+         {this.state.panelElm.map((elm) => {
+         
+             return (<Button value = {elm}  />)
```

Comment:
react requires list of things to have a key, in this case you can use the index of the item as the key.